lib/rototiller/task/params/env_var.rb

```diff
@@ -44,21 +44,25 @@ def initialize(args={}, &block)
       end
 
       # The formatted messages about this EnvVar's status to be displayed to the user
+      # @param indent [String] how far to indent each message
       # @return [String] the EnvVar's message, formatted for color and meaningful to the state of the EnvVar
-      def message
+      INDENT_ARRAY = ["","  ","    ","      ","        ","          "]
+      def message(indent=0)
+        # we only need up to indent 4, really
+        indent = 1 if indent > 1
```

Comment:
Why is this getting reset to one if it is greater than one? Shouldn't it just use the indent passed in?